xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -19,8 +19,18 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides a high-performance API for forward-only, read-only access to the UTF-8 encoded JSON text. It processes the text sequentially with no caching and by default adheres strictly to the <see href="https://tools.ietf.org/html/rfc8259">JSON RFC</see>.</summary>
```

Comment:
Do you think the following should be a remark instead?
```text
It processes the text sequentially with no caching and by default adheres strictly to the <see href="https://tools.ietf.org/html/rfc8259">JSON RFC</see>.
```

Reply:
Yes!